blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md

```diff
@@ -73,219 +73,196 @@ To tokenize a physical asset on the Celo blockchain, you will need to follow the
 
 *Determine the type of asset* you want to tokenize and ensure that it is allowed on the Celo platform. Some examples of assets that can be tokenized on Celo include real estate, precious metals, and works of art.
 
-*Choose the appropriate Ethereum Request for Comment (ERC) standard* for representing your asset on the blockchain. There are several different ERC standards that can be used for tokenizing assets, including ERC-721 and ERC-20.
+*Choose the appropriate Ethereum Request for Comment (ERC) standard* for representing your asset on the blockchain. Several different ERC standards can be used for tokenizing assets, including ERC-721 and ERC-20.
 
 *Create a smart contract on the Celo blockchain* that represents the asset and defines its ownership and transfer rules. This smart contract should specify the conditions under which the asset can be bought, sold, or transferred, as well as any restrictions or limitations on these actions.
 
 ## Different types of ERC can be used to represent different types of assets on Celo.
 
-Ethereum Request for Comment (ERC) standards are technical specifications that define how tokens can be created and used on the Celo blockchain. There are several different ERC standards that can be used for tokenization, including ERC-20, ERC-721, ERC-1400, ERC-3643, and ERC-1404.
+Ethereum Request for Comment (ERC) standards are technical specifications that define how tokens can be created and used on the Celo blockchain. Several different ERC standards can be used for tokenization, including ERC-20, ERC-721, ERC-1400, ERC-3643, and ERC-1404.
 
 **ERC-20** is a widely used standard for creating tokens that represent interchangeable assets, such as commodities or currencies. ERC-20 tokens are created using smart contracts and are stored in Celo wallets. They can be traded on exchanges and used in a variety of applications, such as fundraising through initial coin offerings or representing ownership in a company or project.
 
-OpenZeppelin provides a powerful set of tools to help you create your own token on Ethereum, including the popular ERC20 standard. You can see it on [https://docs.openzeppelin.com/contracts/4.x/wizard](https://docs.openzeppelin.com/contracts/4.x/wizard)
+OpenZeppelin provides a powerful set of tools to help you create your token on Ethereum, including the popular ERC20 standard. You can see it on [https://docs.openzeppelin.com/contracts/4.x/wizard](https://docs.openzeppelin.com/contracts/4.x/wizard)
+Contributor
 
 Here is an example of ERC20
 ```
```

Comment:
add the language type after the backslash for example ```solidity...

Reply:
Done